Movie_MVVM/Movie_MVVM/FilmInfo/FilmInfoViewModel.swift

```diff
@@ -0,0 +1,69 @@
+// FilmInfoViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Вью модель экрана с информацией о выбранном фильме
+final class FilmInfoViewModel: FilmInfoViewModelProtocol {
+    // MARK: - Private properties
+
+    private var networkService: NetworkServicable
+    private var imageService: ImageServicable
+    private var coordinator: SecondCoordinatorProtocol
+
+    // MARK: - Public properties
+
+    var updateActorsHandler: (([Actor]) -> Void)?
+    var updateTrailersHandler: (([Trailer]) -> Void)?
+    var movie: Movie?
+
+    // MARK: - init
+
+    init(
+        networkService: NetworkServicable,
+        imageService: ImageServicable,
+        movie: Movie?,
+        coordinator: SecondCoordinatorProtocol
+    ) {
+        self.networkService = networkService
+        self.movie = movie
+        self.coordinator = coordinator
+        self.imageService = imageService
+    }
+
+    // MARK: - Public methods
+
+    func fetchActors() {
+        guard let id = movie?.currentFilmId else { return }
+        networkService.fetchActors(id: id) { [weak self] result in
+            guard let self = self else { return }
+            switch result {
+            case let .success(actors):
+                self.updateActorsHandler?(actors)
+            case let .failure(error):
+                print(error.localizedDescription)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
error handler добавь